src/skins/vector/css/matrix-react-sdk/structures/_FilePanel.scss

```diff
@@ -15,13 +15,8 @@ limitations under the License.
 */
 
 .mx_FilePanel {
-    -webkit-box-ordinal-group: 2;
```

Comment:
oh, I might have misunderstood what's going on here - these are now automatically added by postcss?

I still wonder if we can have separate PRs: (1) start using postcss; (2) make use of it. But I can see it may not be worth it.